README.md

```diff
@@ -71,6 +71,11 @@ Fully managed hosting of your personal Haven is available too, check out: https:
 
 ## Docker
 
+1. Install `docker` and `docker-compose`
```

Comment:
Let's add links for docker and docker-compose for further details on how to install them.

Reply:
Now It's OK!